### PR TITLE
Fixed status code of 0

### PR DESCRIPTION
**Pull request by stevenpkg**
This fix address issue https://github.com/watson-developer-cloud/assistant-simple/issues/232

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -80,7 +80,8 @@ app.post('/api/message', function (req, res) {
   // Send the input to the assistant service
   assistant.message(payload, function (err, data) {
     if (err) {
-      return res.status(err.code || 500).json(err);
+      var status = (err.code  !== undefined && err.code > 0)? err.code : 500;
```
Comment (germanattanasio) [inline app.js]: You can do 
```
res.status((err.code && err.code > 0) ? err.code : 500).json(err)
```

PLease don't use `var` use `const`

Comment (germanattanasio) [inline app.js]: It's the same comment as https://github.com/watson-developer-cloud/assistant-simple/pull/233#pullrequestreview-227758865

Comment (stevenpkg) [reply app.js]: fixed

Comment (stevenpkg) [reply app.js]: fixed

**Pull request merged**